java/ql/src/Security/CWE/CWE-927/SensitiveResultReceiver.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Leaking sensitive information through a ResultReceiver
+ * @description An Android application obtains a ResultReceiver from a
+ *              third-party component and uses it to send sensitive data
```

Comment:
```suggestion
 * @description Sending sensitive data to a 'ResultReceiver' from an untrusted source
 *              can allow malicious actors access to your information.
```

I rephrased this so it follows the "X causes behavior Y" syntax, but please let me know if I've misunderstood this query!